packages/block-editor/src/utils/transform-styles/transforms/wrap.js

```diff
@@ -27,6 +27,10 @@ const wrap =
 				return selector;
 			}
 
+			if ( selector.trim().startsWith( namespace ) ) {
```

Comment:
Should it be startsWith namespace + ` ` ?

Reply:
Good point